backend/development.ini.sample

```diff
@@ -498,3 +512,10 @@ level=NOTSET
 # number of max log file = 5
 args=('tracim.log','a',10*1024*1024,5)
 formatter=generic
+
+# file log for tracim_notification
```

Comment:
same remark: why not to use default handler by default (the admin sys may change this if he wants to - but default configuration would be simplier)